ddpui/tests/integration_tests/dbt_automation/test_bigquery_ops.py

```
@@ -27,32 +31,140 @@
 
-    wc_client = get_client(
-        "bigquery",
-        json.loads(os.getenv("TEST_BG_SERVICEJSON")),
-        os.environ.get("TEST_BG_LOCATION"),
+    wc_client = WarehouseFactory.connect(
+        TEST_BG_CREDS, "bigquery", os.environ.get("TEST_BG_LOCATION")
     )
     schema = os.environ.get("TEST_BG_DATASET_SRC")  # source schema where the raw data lies
 
     @staticmethod
     def execute_dbt(cmd: str, select_model: str = None):
```

Ruff (0.15.10)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)